vmhooks/baseOps.md

```diff
@@ -383,9 +386,7 @@ module BASEOPS
               => #executeOnDestContext(Dest, 0, Transfers, GasLimit, Func, Args)
                  ...
         </instrs>
-        <callee> Callee </callee>
       requires 0 <Int lengthBytes(Func)
-       andBool #isSmartContract(Callee)
```

Comment:
Why is it reasonable to remove this check?

Reply:
It was both redundant and incorrect. `callContract` will do the same check, so there is no need to have it here. Also, it should be `Dest` not `Callee`. The callee is already a contract since it is currently running.